transposed/transposed.md

```diff
@@ -0,0 +1,265 @@
+so now it's possible to carry out that intent.
+
+## Delaying until after C++26 would be a breaking change
+
+The type of the layout of the `mdspan` returned by `transposed` is observable
+and is specified in the current wording.
+Therefore, delaying this change until after C++26 would be a breaking change.
+
+## What happens if we don't do this?
+
+One possibility is that P1673 implementations will not optimize
+for `layout_left_padded` and `layout_right_padded` `mdspan`.
+That would be unfortunate, because those are exactly the layouts
+that the BLAS Standard supports.
+This would hinder adoption of P1673 algorithms.
+
+Another possibility is that implementations of P1673's algorithms
+could optimize by adding special cases for 
+`layout_transpose<layout_left_padded<PaddingValue>>` and
+`layout_transpose<layout_right_padded<PaddingValue>>` input `mdspan`.
+This should not hinder run-time optimization
+by dispatch to an existing optimized BLAS.
+However, it would complicate the implementation
+and possibly add compile-time cost by introducing
+more internal overloads and/or specializations.
+Furthermore, it would have the same effects for users
+who want to use `transposed` with their own
+P1673-like linear algebra algorithms.
+
+# Alternative: `transposed_mapping` customization point
+
+In the previous section, we mentioned that users
+may want to use `transposed` with their own
+P1673-like linear algebra algorithms.
+A reviewer suggested that we make it possible for users
+to optimize `transposed` for their user-defined layouts,
+by adding a `transposed_mapping` customization point.
+This would be analogous to the `submdspan_mapping` customization point
+that approval of P2630R4 (`submdspan`) added to the C++ Working Draft.
+This design would have the following advantages.
+
+1. It would be easier to specify the wording of `transposed`.
+    Instead of its current list of special cases,
+    it would look more like the `submdspan` wording
+    that puts all the layout-specific behavior in the customization point.
+
+2. Implementations that provide implementation-specific layouts
+    could optimize `transposed` for those layouts.
+
+3. Users could use `transposed` with their custom P1673-like algorithms
+    and implement optimizations for their user-defined layouts.
+
+Here are some reasons why WG21 might _not_ want to do this.
+
+1. It would reserve yet another customization point name.
+
+2. It would not help P1673 implementations optimize for user-defined layouts.
+
+3. `submdspan_mapping` enables functionality --
+    the ability to slice `mdspan` with user-defined layouts --
+    while `transposed_mapping` would only enable (or simplify) optimizations.
+
+4. LEWG has already seen the proposed design over several reviews
+    (the last being the 2022/07/05 telecon review of P1673R9),
+    but has not yet had a chance to review
+    this alternative customization point design.
+
+We would like LEWG to poll this design alternative.
```

Comment:
Would it be possible to add `transposed_mapping` in the future even if it wasn't adopted for this paper?

Reply:
Hi @nmm0 !  It's possible in theory.  In practice, though, introducing `transposed_mapping` after C++26 would be a breaking change, as it would change the return type of `transposed` for user-defined layouts.

I personally prefer the `transposed_mapping` design, even if we only use it as a way to simplify wording.  However, WG21 may not like adding a customization point that will be rarely used, for a feature that is not strictly necessary in order to use or even to optimize std::linalg.

Reply:
Thinking harder about `transposed_mapping` as a customization point made me change my mind; I'm now recommending against it.  I'll explain this in an updated draft.